snippets/change-streams-monitor/package.json

```
@@ -0,0 +1,14 @@
+{
+  "name": "@edmallia/snippet-change-stream-monitor",
```

```suggestion
  "name": "@mongosh/snippet-change-stream-monitor",
```

This needs to be in a namespace that our automatic tooling has push access for :)